### PR TITLE
Announcing Izzy - Istio's CNCF Phippy family Character

### PR DESCRIPTION
**Pull request by kfaseela**

### CODE REVIEW

**Comment by kfaseela** [comment]
Adding a dnm so that this can be published at the right time

**Review by keithmattix — APPROVED** [review]
LGTM!

**Review by linsun — APPROVED** [review]

**Comment by linsun** [comment]
Thanks Faseela!

**Comment by craigbox** [comment]
https://deploy-preview-14710--preliminary-istio.netlify.app/latest/blog/2024/istio-phippy/

File: content/en/blog/2024/istio-phippy/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Istio joins Phippy and friends - Welcome Izzy!"
+description: Announcing Istio's new Phippy character.
```
Comment (craigbox) [inline content/en/blog/2024/istio-phippy/index.md:3]:
```suggestion
title: "Istio joins Phippy and friends — Welcome Izzy!"
description: Announcing the latest member of the CNCF family of mascots.
```
to not say "Phippy" twice

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by kfaseela** [comment]
@craigbox if the PR looks good now, you can approve. I will remove the dnm at the announcement time

**Review by craigbox — APPROVED** [review]
LGTM, thanks!

**Review by kebe7jun — APPROVED** [review]

File: content/en/blog/2024/istio-phippy/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Having sailed into, and proudly graduated within the Cloud Native Computing Foundation in 2023, it is now time for Istio to join the [CNCF Phippy family’s](https://www.cncf.io/phippy/) mission to demystify and simplify cloud native computing.
+
+The Istio Steering Committee is excited to unveil Izzy Dolphin, the Istio Indo-Pacific Bottlenose, who today dives into the family of "Phippy and Friends".
+
+{{< image width="75%"
+    link="./Izzy.png"
+    alt="Izzy, the Istio Dolphin"
+    >}}
+
+Istio stands on the shoulders of several other CNCF projects, including Kubernetes, Envoy, Prometheus, and Helm. Izzy is proud to join Phippy, Hazel and Captain Kube’s gang, taking cloud native to the masses.
```
Comment (ericvn) [inline content/en/blog/2024/istio-phippy/index.md:18]: Inconsistent Oxford comma?
```suggestion
Istio stands on the shoulders of several other CNCF projects, including Kubernetes, Envoy, Prometheus, and Helm. Izzy is proud to join Phippy, Hazel, and Captain Kube’s gang, taking cloud native to the masses.
```

**Review by ericvn — APPROVED** [review]

**Review by irisdingbj — APPROVED** [review]

File: content/en/blog/2024/istio-phippy/index.md (hunk 2 above)
Comment (craigbox) [reply content/en/blog/2024/istio-phippy/index.md]: ❤️ 🔥

**Comment by Charley-Mann** [comment]
Hey folks FYI we are ready and set with the CNCF blog announcement for 09:00 PST today (March 13), as well as follow-up social media posts ^ @kfaseela

**Comment by kfaseela** [comment]
> Hey folks FYI we are ready and set with the CNCF blog announcement for 09:00 PST today (March 13), as well as follow-up social media posts ^ @kfaseela 

Hello Charley, looks like the blog was already published yesterday.. Daniel crook sent me the link https://www.cncf.io/blog/2024/03/11/istio-joins-phippy-and-friends-welcome-izzy/

**Pull request merged**